compiler/daml-lf-ast/test/DA/Daml/LF/Ast/Tests.hs

```diff
@@ -63,6 +64,17 @@ substitutionTests = testGroup "substitution"
                 TBuiltin BTArrow `TApp` vBeta11 `TApp` vBeta1
             ty2 = substitute subst ty1
         assertBool "wrong substitution" (alphaEquiv ty1 ty2)
+
+    , testCase "freeVars/TypeLevelNat" $ do
+        let -- no thank you HLint
```

Comment:
What is this about?

Reply:
Adding a lone `let` prevents HLint complaining about the unnecessary parens, which were added in the testcase to improve layout. But I removed these parens now.

Reply:
thanks!